src/pyhf/contrib/viz/brazil.py

```diff
@@ -48,3 +48,93 @@ def plot_results(ax, mutests, tests, test_size=0.05):
 
     ax.set_xlabel(r"$\mu$ (POI)")
     ax.set_ylabel(r"$\mathrm{CL}_{s}$")
+
+
+def plot_cls_components(
```

Comment:
@kratsg @lukasheinrich This is still WIP, but early feedback on API is very welcomed.

Reply:
there should be an internal function call `plot_pvalue()` which is just an elided version of `ax.plot` for the particular CL value being plotted -- since you're essentially doing the same thing for CLs, CLs+b, CLb.

Reply:
Can you elaborate further? I'm not really sure what this gets us as these are all just calls to `ax.plot(x, y)` with different values of `y`.